src/client/rspamc.cxx

```diff
@@ -1030,8 +1030,14 @@ rspamc_symbols_output(FILE *out, ucl_object_t *obj)
 		const ucl_object_t *cmesg;
 
 		while ((cmesg = ucl_object_iterate (elt, &mit, true)) != nullptr) {
-			fmt::print(out, "Message - {}: {}\n",
-					ucl_object_key(cmesg), ucl_object_tostring(cmesg));
+			if (cmesg->type == UCL_STRING) {
+				fmt::print(out, "Message - {}: {}\n",
+						ucl_object_key(cmesg), ucl_object_tostring(cmesg));
+			} else {
+				fmt::print(out, "Message - {}: {}\n",
+						ucl_object_key(cmesg),
+						ucl_object_emit(cmesg, UCL_EMIT_JSON_COMPACT));
```

Comment:
We have discussed that with @citrin recently, and I would also suggest to make it in a different way.

1) When you do `ucl_object_emit` you allocate memory that is never freed and that's a memory leak
2) `cmesg` can be lengthy and not fitting into the terminal, so I suggest to call emit first, then check the length of the string and truncate it if the length is more than ~60 characters. We are talking about human readable representation after all.